### ISSUE TITLE
Add sun card icon to HACS

### ISSUE THREAD

**Issue by ThomDietrich** [rejuvenate/lovelace-sun-card#10]
It would be great to add an Icon for this card within HACS:

![](https://user-images.githubusercontent.com/2870104/225671830-75699cb0-98e3-4181-81cb-2d4ffc9f32bb.png)

**Comment by raetha** [comment]
I'll second this. HACS was only showing the now-archived AitorDB version. When trying to add this as a custom repository, an error is given that the "Repository structure for main is not compliant".

**Comment by ThomDietrich** [comment]
Hey @raetha, you might be mistaken about the meaning of this issue. It's about showing an icon to the left to the entry "Sun Card".

> When trying to add this as a custom repository

Please give us a few more days to sort some things out and register our repository with HACS. After that you will be able to install Sun Card as before, plus you might see some cool additional features soon.

**Comment by raetha** [comment]
@ThomDietrich, you are totally right! I found this yesterday when HACS reported the old one as archived. I didn’t realize at the time this repo was brand new, and I just tried to add it to HACS manually to get an updated card. Sorry to add confusion, and glad you and others have picked this up to run with it. I wish you the best getting things going!

**Comment by ThomDietrich** [comment]
https://github.com/hacs/integration/issues/3095#issuecomment-1475159884

**Comment by jfreves** [comment]
Is it suppose to show the new integration in HACS already?

**Comment by ThomDietrich** [comment]
Wrong ticket. Please see #1 and also our README: https://github.com/rejuvenate/lovelace-horizon-card#hacs
